### ISSUE TITLE
checkpoints failed loading 

### ISSUE THREAD

**Issue by xuyuxiu83** [nianticlabs/doubletake#5]
I'm so sure I have downloaded the checkpoints from your link and put it in the right place. But the output makes me so confused and it shows that I failed to load model normally:
/home/xyx/mambaforge/envs/doubletake/bin/python /home/xyx/stereo_matching/codes/doubletake-main/src/doubletake/test_offline_two_pass.py --name doubletake_offline --output_base_path /home/xyx/stereo_matching/codes/doubletake-main/outputs/vdr --config_file /home/xyx/stereo_matching/codes/doubletake-main/configs/models/doubletake_model.yaml --load_weights_from_checkpoint /home/xyx/stereo_matching/codes/doubletake-main/weights/doubletake_model.ckpt --data_config /home/xyx/stereo_matching/codes/doubletake-main/configs/data/vdr/vdr_default_offline.yaml --num_workers 8 --batch_size 2 --fast_cost_volume --run_fusion --depth_fuser custom_open3d --fuse_color --fusion_max_depth 3.5 --fusion_resolution 0.02 --trim_tsdf_using_confience --extended_neg_truncation --dump_depth_visualization
Using /home/xyx/.cache/torch_extensions/py39_cu117 as PyTorch extensions root...
Detected CUDA files, patching ldflags
Emitting ninja build file /home/xyx/.cache/torch_extensions/py39_cu117/ext/build.ninja...
Building extension module ext...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module ext...
Seed set to 42
########################### Options ###########################

     random_seed: 0
     name: doubletake_offline
     log_dir: /home/xyx/tmp/tensorboard
     notes: 
     log_interval: 100
     val_interval: 1000
     val_batches: 100
     dataset: vdr
     dataset_path: /home/xyx/stereo_matching/codes/doubletake-main/datasets/vdr
     num_workers: 8
     tuple_info_file_location: /home/xyx/stereo_matching/codes/doubletake-main/data_splits/vdr/
     mv_tuple_file_suffix: _eight_view_deepvmvs_offline.txt
     frame_tuple_type: offline
     model_num_views: 8
     num_images_in_tuple: 8
     dataset_scan_split_file: /home/xyx/stereo_matching/codes/doubletake-main/data_splits/vdr/scans.txt
     split: test
     image_width: 512
     image_height: 384
     shuffle_tuple: False
     test_keyframe_buffer_size: 30
     rotate_images: False
     lr: 0.0001
     wd: 0.0001
     num_sanity_val_steps: 0
     max_steps: 110000
     batch_size: 2
     val_batch_size: 16
     gpus: 2
     precision: 16
     lr_steps: [70000, 80000]
     resume: None
     load_weights_from_checkpoint: /home/xyx/stereo_matching/codes/doubletake-main/weights/doubletake_model.ckpt
     lazy_load_weights_from_checkpoint: None
     image_encoder_name: efficientnet
     depth_decoder_name: unet_pp
     loss_type: log_l1
     matching_encoder_type: resnet
     matching_feature_dims: 16
     matching_scale: 1
     matching_num_depth_bins: 64
     min_matching_depth: 0.25
     max_matching_depth: 5.0
     cv_encoder_type: multi_scale_encoder
     feature_volume_type: mlp_mesh_hint_feature_volume
     output_base_path: /home/xyx/stereo_matching/codes/doubletake-main/outputs/vdr
     run_fusion: True
     fuse_color: True
     fusion_max_depth: 3.5
     fusion_resolution: 0.02
     depth_fuser: custom_open3d
     trim_tsdf_using_confience: True
     extended_neg_truncation: True
     single_debug_scan_id: None
     skip_frames: None
     skip_to_frame: None
     mask_pred_depth: False
     cache_depths: False
     fusion_use_raw_lowest_cost: False
     high_res_validation: False
     fast_cost_volume: True
     standard_fps: 30
     dump_depth_visualization: True
     use_precomputed_partial_meshes: False
     viz_render_width: 640
     viz_render_height: 480
     cam_marker_size: 0.7
     back_face_alpha: 0.5
     viz_fixed_min_max: False
     fill_depth_hints: True
     depth_hint_aug: 0.5
     depth_hint_dir: None
     load_empty_hint: False

###############################################################

################################################################################
################################################################################

################################################################################
##################### Running fusion! Using custom_open3d ######################
Output directory:
/home/xyx/stereo_matching/codes/doubletake-main/outputs/vdr/doubletake_offline/vdr/offline/meshes/0.02_3.5_custom_open3d_color_neg_trunc_weight_trimmed 
################################################################################

################################################################################
############################### Saving quick viz.###############################
Output directory:
/home/xyx/stereo_matching/codes/doubletake-main/outputs/vdr/doubletake_offline/vdr/offline/viz/quick_viz 
################################################################################

opts.load_weights_from_checkpoint /home/xyx/stereo_matching/codes/doubletake-main/weights/doubletake_model.ckpt
2024-09-18 16:12:58.486 | INFO     | doubletake.utils.model_utils:load_model_inference:27 - Failed to load model normally. Using manual loading via state_dict.
